.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: scala
+sudo: false
+jdk: oraclejdk8
+
+scala:
+  - 2.10.6
+
+script:
+  - sbt "++$TRAVIS_SCALA_VERSION" publishLocal
+  - sbt "++$TRAVIS_SCALA_VERSION" test
+  - sbt "++$TRAVIS_SCALA_VERSION" scripted
+
```

Comment:
What would be the best way to enable this for all versions? I'm asking as I never figured out an elegant way to run all Laika tests with a single command. I always ran:

```
project laika-core
++ test
project laika-pdf
++ test
project laika-sbt
scripted
```
simply because running `++ test` on the top level produced errors in the laika-sbt project and overriding crossScalaVersions in a sub-project gets ignored.